arch/arm/boot/dts/imx6ul-ts7180.dts

```diff
@@ -0,0 +1,778 @@
+// SPDX-License-Identifier: GPL-2.0+ or X11
+/*
+ * Copyright (C) 2021-2023 Technologic Systems, Inc. dba embeddedTS
+ */
+
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+#include "imx6ul.dtsi"
+
+/ {
+	model = "embeddedTS i.MX6ul TS-7180";
+	compatible = "technologic,ts7180", "fsl,imx6ul";
+
+	aliases {
+		ethernet0 = &fec2;
+		ethernet1 = &fec1;
+	};
+
+	chosen {
+		stdout-path = &uart1;
+	};
+
+	memory@80000000 {
+                device_type = "memory";
+		reg = <0x80000000 0x20000000>;
```

Comment:
Since this can be configured with two sizes of RAM, the length should not be specified and would be filled in by U-Boot. Common notation for this is:
```
                /* Memory size to be filled in by U-Boot */
                reg = <0x80000000 0>;
```